README.md

```diff
@@ -55,7 +55,7 @@ Parameters:
  * `extra`: ( optional ) string, Client provided metadata that get passed back in the callback.
  * `mime_type`: ( optional ) string, series of mime types that the download is going to be verified against.
  * `download_timeout`: ( optional ) int, HTTP client timeout per Job, in seconds.
- * `user_agent`: ( optional ) string, User-Agent request header per Job.
+ * `request_headers`: ( optional ) map(i.e hash/dictionary), HTTP Request Headers per job.
```

Comment:
type is `object` here. We could use the notation `object{string=>string}` for clarity.

Reply:
Done!